tilequeue/worker.py

```diff
@@ -189,59 +220,69 @@ def __call__(self, stop):
 
         while not stop.is_set():
             try:
-                data = self.input_queue.get(timeout=timeout_seconds)
+                all_data = self.input_queue.get(timeout=timeout_seconds)
             except Queue.Empty:
                 continue
-            if data is None:
+            if all_data is None:
                 saw_sentinel = True
                 break
 
-            coord = data['coord']
-            nominal_zoom = coord.zoom + self.metatile_zoom
-            unpadded_bounds = coord_to_mercator_bounds(coord)
+            for fetch, data in self.fetcher.start(all_data):
```

Comment:
Is `start` the best name for it? Seems like it's more of a `group` or `parcel` kind of operation. Names are hard :)

Reply:
You're right, `start` is a pretty awful name for it. Perhaps it fitted better back when it was a `@contextmanager`, but now it's `yield`ing a sequence of `(fetch, data)` tuples then it's not an intuitive name for it.

Would something that indicates iteration be better? Like `items()`, `each()` or `fetch_tiles()`?

Reply:
Yup, I like any of those names.

Reply:
Cool, thanks. Fixed in 5840e9a.